example/demo/src/ui-controls/ui.tsx

```diff
@@ -56,94 +41,50 @@ function UI() {
 }
 
 function FormExample() {
-  const userName = localStateExperimental('', 'form/userName')
-  const notification = localStateExperimental('', 'form/notification')
-  const onFormSubmit = event(
-    (state) => {
-      console.log(`Values: ${state}`)
-      return response(null)
-        .action(setStateAction(userName, ''))
-        .action(
-          setStateAction(
-            notification,
-            'Thank you for submitting the form! Check the logs on the backend to see the submitted values'
-          )
-        )
-    },
-    { args: { userName } }
-  )
   return (
-    <YStack gap="$8" padding="$4">
-      <YStack gap="$4">
-        <H4>Simple form</H4>
-        <Form onSubmit={onFormSubmit}>
-          <YStack gap="$4">
-            <Text>User name</Text>
-            <Input
-              value={userName}
-              onChangeText={[setStateAction(userName), setStateAction(notification, '')]}
-            />
-          </YStack>
-          <Text>{notification}</Text>
-          <Button onPress={onFormSubmit}>Submit form</Button>
-        </Form>
-      </YStack>
-    </YStack>
-  )
-}
-
-function SliderExample() {
-  const slider = localStateExperimental(0, 'slider/slider')
-
-  const sliderRangeStart = localStateExperimental(25, 'slider/range-start')
-  const sliderRangeEnd = localStateExperimental(75, 'slider/range-end')
-  const sliderWithReset = localStateExperimental(0, 'slider/with-reset')
-
-  return (
-    <YStack gap="$8" padding="$4">
-      <YStack gap="$4">
-        <H4>Single slider</H4>
-        <SliderField
-          value={[slider]}
-          onValueChange={setStateAction(slider, eventPayload([0, 0]))}
+    <ScrollView contentContainerStyle={{ padding: '$4' }}>
+      <RiseForm
+        onSubmit={(values) => {
+          console.log('Form submitted', values)
+          return response(null)
+            .action(toast('Thank you for submitting your feedback'))
+            .action(goBack())
+        }}
+      >
+        <InputField id="name" label="Name" />
+        <TextField id="feedback" label="Feedback" />
+        <CheckboxField id="subscribe" label="Subscribe to newsletter" defaultChecked={true} />
+        <SliderField id="rating" label="Rating" defaultValue={[20, 60]} />
+        <SwitchField id="anonymous" label="I want to be anonymous" />
+        <SelectField
```

Comment:
Shall we keep the form example like this? @ericvicenti 

Or... 
- Input
- Input with custom label (you can render string or pass custom element)
- TextField
- Slider
- Range Slider
- Switch
- Toggle Group Horizontal
- Toggle Group Vertical
- Toggle Multiple
- Checkbox
- Radio Group

Reply:
I am going to rewrite it to showcase all different types instead. Improvements are welcome!